l1-contracts/deploy-scripts/DeployUtils.s.sol

```diff
@@ -136,6 +136,7 @@ struct TokensConfig {
 // solhint-disable-next-line gas-struct-packing
 struct GeneratedData {
     bytes forceDeploymentsData;
+    bytes forceDeploymentsDataGW;
```

Comment:
instead of having it like this, maybe it makes more sense to have another instance of GeneratedData for GW, i.e. GWGeneratedData? If we start adding new field it will scale better and we keep DeployUtils for a single chain